github_api/misc.py

```diff
@@ -1,5 +1,6 @@
 from math import log
 
+
```

Comment:
It was #344 here is the [status](https://api.github.com/repos/chaosbot/Chaos/commits/cea1616848f9182b82a0354feceecaa0e1bdc2fb/status), as you can see - it's only from chaos